code/modules/vore/fluffstuff/custom_guns_vr.dm

```diff
@@ -412,6 +412,32 @@
 	desc = "Some serious drywall work, coming up!"
 */
 
+//hzdonut:Jesse Soemmer
+obj/item/weapon/gun/projectile/revolver/lemat/fluff/jesselemat
```

Comment:
This looks like just a copy of the lemat. Most of these vars will inherit from the parent object and don't need to be defined here. MOST NOTABLY there's nothing here that I can see that overrides the lemat's secondary fire modes, max secondary ammo, or anything else.

Either override the lemat-specific functions for the fluff version, or make it a child of the plain revolver.